modules/geo-layers/src/h3-layers/h3-hexagon-layer.js

```diff
@@ -155,8 +155,10 @@ export default class H3HexagonLayer extends CompositeLayer {
 
   _shouldUseHighPrecision() {
     const {resolution, hasPentagon, hasMultipleRes} = this.state;
+    const {viewport} = this.context;
     return (
       this.props.highPrecision ||
+      viewport.resolution ||
```

Comment:
Looks like we retrigger a calculation every time the layer state is updated.

But what happens when there are multiple viewports?

Reply:
This layer does not support multi-view (same thing for HexagonLayer)